### PR TITLE
docs(microservices): updated documented grpc sample

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: #1776

## What is the new behavior?
N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This PR, updated documented grpc sample

### CODE REVIEW

**Review by jmcdo29 — APPROVED** [review]
Overall I'd say it looks good and reads well. Personally I'd prefer to keep these files in the project root, but this approach works too.

File: content/microservices/grpc.md [hunk 1]
```diff
@@ -233,6 +233,16 @@ imports: [
...
+Later in the `nest-cli.json` file, we add the assets property that allows us to distribute non-TypeScript files, in our case the `.proto` file. This way, during the build phase, you can edit non-TypeScript files and compile and test iteratively.
+
+```json
+{
+  "compilerOptions": {
+    "assets": ["**/*.proto"]
+  }
+}
+```
+
```
Comment (kamilmysliwiec) [inline content/microservices/grpc.md:245]: This addition looks great overall, but I don't that "Client" heading is an appropriate section for this (especially in the middle of the `ClientsModule` example which is completely unrelated to this newly inserted block).

Comment (Tony133) [reply content/microservices/grpc.md:245]: What if we put it before the subtitle "Client"?

Comment (kamilmysliwiec) [reply content/microservices/grpc.md:245]: I think the best place to add it would be the "Overview" section (by the end of it) @Tony133

Comment (Tony133) [reply content/microservices/grpc.md]: Moved to the "Overview" section as you said

File: content/microservices/grpc.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ const app = await NestFactory.createMicroservice(AppModule, {
...
 
+Later in the `nest-cli.json` file, we add the assets property that allows us to distribute non-TypeScript files, in our case the `.proto` file. This way, during the build phase, you can edit non-TypeScript files and compile and test iteratively.
+
+```json
+{
+  "compilerOptions": {
+    "assets": ["**/*.proto"]
+  }
+}
+```
```
Comment (kamilmysliwiec) [inline content/microservices/grpc.md:50]:
```suggestion
In the `nest-cli.json` file, we add the `assets` property that allows us to distribute non-TypeScript files, and `watchAssets` - to turn on watching all non-TypeScript assets. In our case, we want `.proto` files to be automatically copied to the `dist` folder. 

```json
{
  "compilerOptions": {
    "assets": ["**/*.proto"],
    "watchAssets": true
  }
}
```
```

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**